R/taxonomy.R

```diff
@@ -209,6 +208,19 @@ setMethod("checkTaxonomy", signature = c(x = "SummarizedExperiment"),
     }
 )
 
+#'@export
+# Function to set taxonomy ranks
+setTaxonomyRanks <- function(rank, namespace = "mia") {
+    rank <- tolower(rank)
+    assignInNamespace("TAXONOMY_RANKS", rank, ns = asNamespace(namespace))
+    cat("Taxonomy ranks set to:", rank, "\n")
```

Comment:
Printing is not necessary, because user knows that he/she has changed the ranks

(moreover printing (print(), cat()) is not recommended way to print, because user cannot disable those. Preferable way is to use message(), warning(), or stop()